README.md

```diff
@@ -66,11 +66,29 @@ class Resource(path: String) {
 To setup resources correctly and avoid `FilNotFoundException` & co:
 
 1. **Put them in the [resources folder](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.SourceSet.html#org.gradle.api.tasks.SourceSet:resources) of a source set.** For example, `src/commonTest/resources/` or `src/jsTest/resources/`.
-2. **Specify the path relative to the project's directory.** For example, `src/commonTest/resources/a-folder/a-file.txt`.
-
+2. **Specify the path relative to the resources folder.** For example, if your file is at
+   `src/commonTest/resources/a-folder/a-file.txt`, use `Resource("a-folder/a-file.txt")`.
 
 With these in mind, you're ready to go.
 
+### Collision handling
+
+When running platform-specific tests, the platform-specific resource takes precedence if it exists.
+
+If a resource with the same path also exists in `commonTest`, Gradle will fail the build with the following error:
+`Entry 'your_file' is a duplicate but no duplicate handling strategy has been set.`
+
+This behavior forces explicit resolution of duplicates between platform-specific and common resources.
+
```

Comment:
```suggestion
```

This isn't implemented, so let's wait for it to be available before we document it.